packages/base/room.gts

```diff
@@ -226,70 +208,12 @@ class EmbeddedMessageField extends Component<typeof MessageField> {
   }
 
   get cardComponent() {
-    if (!this.attachedCard || isMatrixCardError(this.attachedCard)) {
+    let card = this.attachedCardResource?.card;
+    if (!card) {
       return;
     }
-    return this.attachedCard.constructor.getComponent(
-      this.attachedCard,
-      'isolated',
-    );
+    return card.constructor.getComponent(card, 'isolated');
   }
-
-  get cardError() {
-    if (isMatrixCardError(this.attachedCard)) {
-      return this.attachedCard;
-    }
-    return;
-  }
-
-  loadAttachedCard = restartableTask(async () => {
```

Comment:
This turned out to be basically a bespoke implementation of the card resource that we were able to eliminate in facvor of an actual resource